### DISCUSSION TITLE
How can I set up AIO to use my discrete GPU for tensorflow operations?

### DISCUSSION THREAD

**Discussion by ixanza** [nextcloud/all-in-one#2082] [Questions] [answered]
I run AIO on a laptop homeserver with an intel iGPU and an nvidia discrete GPU using unRAID.

I'd like to use the Recognize app and I understand using the GPU can speed up recognition, how would I set this up?

**Comment by szaimen** [comment, marked as answer]
Hi, you cannot while https://github.com/nextcloud/recognize/issues/73 is not implemented. After that is done, we can look into putting the recognize dependencies into a separate container which should also allow recognize to use the dedicated gpu.
